docs/guides/converting_and_upgrading/optax_update_guide.rst

```diff
@@ -112,13 +111,11 @@ generic building blocks.
       optax.scale(-learning_rate),
   )
 
-Weight Decay
+Weight decay
 ------------
 
-Some of Flax's optimizers also include a weight decay. In Optax, some optimizers
-also have a weight decay parameter (such as |optax.adamw()|_), and to others the
-weight decay can be added as another "gradient transformation"
-|optax.add_decayed_weights()|_ that adds an update derived from the parameters.
+- Some of `flax.optim` optimizers include the weight decay parameter.
```

Comment:
In the docs page, `flax.optim` is rendered as italicized instead of code font. I think you need to use double back quotes, i.e. \``flax.optim``